client/src/store/graphql/files/actions.tsx

```diff
@@ -0,0 +1,77 @@
+import { useLazyResultQuery } from 'store/graphql/utils/query';
+import { gql } from '@apollo/client';
+
+export type QueryVariable = {
+  patientId: string;
+};
+
+export const getFilesData = (variables: QueryVariable) => () => {
+  const test = gql`{
```

Comment:
Maybe call this constant 'query'. Makes more sense that 'test'.

Reply:
Agreed. Moreover, maybe it should be in a file of it's own. but if it s the only graphql query so far you can let it here for now.